splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -1203,7 +1202,7 @@ public void addMinRetentionPeriodColumn(TransactionController tc) throws Standar
                     SYSTABLESRowFactory.MIN_RETENTION_PERIOD));
 
             // now upgrade the views if necessary
-            createOrUpdateSystemView(tc, "SYSVW", SYSTABLESRowFactory.SYSTABLE_VIEW_NAME);
+            createOrUpdateSystemView(tc, "SYSVW", "SYSCOLUMNSVIEW");
```

Comment:
Was this a bug? Great that you found it.

The `SYSCOLUMNVIEW` name is probably also available from `SYSCOLUMNSRowFactory`. Can you also check it? We want to avoid hard coded names as much as possible.